### PR TITLE
When reportDiagnostic is Default return descriptor default severity

### LINKED ISSUE #1073: Error in command "roslynator analyze"
**Product and Version Used**: 
roslynator.dotnet.cli   "version": "0.5.0

**Steps to Reproduce**:
roslynator analyze

**Actual Behavior**:
```csharp
```

One or more errors occurred. (Unknown value 'Default'. (Parameter 'reportDiagnostic'))
  Inner exception: Unknown value 'Default'. (Parameter 'reportDiagnostic')

**Expected Behavior**:
```csharp
```

### PR DESCRIPTION
**Pull request by jsliwinski**
fixes #1073

### CODE REVIEW

**Comment by josefpihrt** [comment]
Hi,

Thanks for the PR. I'm able to reproduce the bug.

File: src/Core/Extensions/DiagnosticsExtensions.cs [hunk 1]
```diff
@@ -488,7 +488,11 @@ internal static bool IsEffective(this DiagnosticDescriptor descriptor, SyntaxNod
             }
 
             if (compilationOptions.SpecificDiagnosticOptions.TryGetValue(descriptor.Id, out ReportDiagnostic reportDiagnostic))
-                return reportDiagnostic;
+            {
+                return (reportDiagnostic == Microsoft.CodeAnalysis.ReportDiagnostic.Default)
```
Comment (josefpihrt) [inline src/Core/Extensions/DiagnosticsExtensions.cs:492]: I just verified that not only this condition can return `Default` but also the one above and below. I would suggest to put these three `if`s to separate local method.

**Review by josefpihrt — CHANGES_REQUESTED** [review]
Please update changelog.

**Comment by jsliwinski** [comment]
@dotnet-policy-service agree

**Review by josefpihrt — APPROVED** [review]

**Comment by josefpihrt** [comment]
@jsliwinski Thank you for the contribution!

**Pull request merged**